samza/notification-job/src/main/java/org/sunbird/jobs/samza/service/NotificationService.java

```diff
@@ -31,66 +37,80 @@
 	private JobLogger Logger = new JobLogger(NotificationService.class);
 	private Config appConfig = null;
 	private String accountKey = null;
-	private String msgAuthKey = null;
+	private String smsAuthKey = null;
+	private String smsDefaultSender = null;
 	private ISmsProvider smsProvider = null;
-	private String[] mandatoryParams = null;
+	private IEmailFactory emailFactory = null;
+	private IEmailService emailService = null;
 	private IFCMNotificationService ifcmNotificationService = NotificationFactory
 			.getInstance(NotificationFactory.instanceType.httpClinet.name());
-	private static final String FCM_ACCOUNT_KEY = "fcm_account_key";
-	private static final String MSG_AUTH_KEY = "msg_auth_key";
 
 	public void initialize(Config config) throws Exception {
 		JSONUtils.loadProperties(config);
 		appConfig = config;
-		mandatoryParams = new String[]{
-						Constant.ACTION_NAME, Constant.IDS, Constant.EDATA
-		};
-		accountKey = appConfig.get(FCM_ACCOUNT_KEY);
-		msgAuthKey = appConfig.get(MSG_AUTH_KEY);
-		SMSConfig smsConfig = new SMSConfig(msgAuthKey, "");
+		accountKey = appConfig.get(NotificationEnum.fcm_account_key.name());
+		smsAuthKey = appConfig.get(NotificationEnum.sms_auth_key.name());
+		smsDefaultSender = appConfig.get(NotificationEnum.sms_default_sender.name());
+		SMSConfig smsConfig = new SMSConfig(smsAuthKey, smsDefaultSender);
```

Comment:
new SMSConfig(smsAuthKey, smsDefaultSender); to new SMSConfig(appConfig.get(NotificationEnum.sms_auth_key.name()), appConfig.get(NotificationEnum.sms_default_sender.name()));

Reply:
Done